user/links/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "links"
+pkgver = "2.30"
+pkgrel = 0
+build_style = "configure"
+configure_args = ["--prefix=/usr", "--mandir=PREFIX/share/man"]
+hostmakedepends = [
+    "libjpeg-turbo-devel",
+    "libtiff-devel",
+    "openssl3-devel",
+    "zlib-ng-compat-devel",
+]
+pkgdesc = "Lynx-like text WWW browser"
+license = "GPL-2.0-or-later"
+url = "http://links.twibright.com"
+source = f"{url}/download/{pkgname}-{pkgver}.tar.gz"
```

Comment:
we don't do `{pkgname}` in `source` anymore
```suggestion
source = f"{url}/download/links-{pkgver}.tar.gz"
```